travis-linux.sh

```diff
@@ -27,7 +27,10 @@ if [[ ! -e wx-install-cache/bin/wx-config ]]; then
     if [[ "8d98975eb9f81036261c0643755b98e4bb5ab776" != $(sha1sum wxWidgets-3.1.1.7z | cut -f1 -d' ') ]] ; then exit 1 ; fi
     7z x -o"wxWidgets" -y wxWidgets-3.1.1.7z > /dev/null
     cd wxWidgets || exit 1
-    #patch -p0 < ../patches/wxWidgets/*.patch || exit 1
```

Comment:
@kduske Because this line was commented out, we weren't applying `textentry_gtk_suppressevents.patch` on our official .deb/.rpm builds of TB. I think that was my mistake, I commented it out at a time when there were no patches to apply on Linux, sorry about that.

I forget what the patch was for, but assume your personal dev builds have been with the patch, so it should be safe to start using that patch?

Reply:
This patch fixes https://github.com/kduske/TrenchBroom/issues/2396

Reply:
So yeah, let's use it ;-)